Description: add missing #ifdefs for an ATK enum value that requires a version of ATK newer than what's in bionic
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/ui/accessibility/platform/ax_platform_node_auralinux.cc
+++ b/ui/accessibility/platform/ax_platform_node_auralinux.cc
@@ -3745,11 +3745,13 @@ gfx::Rect AXPlatformNodeAuraLinux::GetEx
       extents.Offset(window_origin);
       break;
     }
+#if defined(ATK_230)
     case ATK_XY_PARENT: {
       gfx::Vector2d parent_origin = -GetParentOriginInScreenCoordinates();
       extents.Offset(parent_origin);
       break;
     }
+#endif
   }
 
   return extents;
@@ -4387,8 +4389,10 @@ gfx::Point AXPlatformNodeAuraLinux::Conv
   switch (atk_coord_type) {
     case ATK_XY_WINDOW:
       return point + GetParentFrameOriginInScreenCoordinates();
+#if defined(ATK_230)
     case ATK_XY_PARENT:
       return point + GetParentOriginInScreenCoordinates();
+#endif
     case ATK_XY_SCREEN:
     default:
       return point;
